packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -39,7 +39,7 @@ const SubjectViewer = types
 
   .views(self => ({
     get disableImageToolbar () {
-      const subject = getRoot(self).subjects.active
+      const subject = tryReference(() => getRoot(self).subjects?.active)
```

Comment:
Question: Just for my understanding, what does the change here implement?

Reply:
`subjects.active` is a reference (it's actually the subject ID.) References should be accessed via `tryReference` to avoid errors. 

https://mobx-state-tree.js.org/API/#tryreference

Reply:
The mock store errored on this line, because `subjects.active` isn't always a valid reference.